### PR TITLE
feat(RDS): add rds read replica instance params

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  fix read replica instance params
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix read replica instance params
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/rds/' TESTARGS='-run TestAccReadReplicaInstance_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/rds/ -v -run TestAccReadReplicaInstance_ -timeout 360m -parallel 4
=== RUN   TestAccReadReplicaInstance_basic
=== PAUSE TestAccReadReplicaInstance_basic
=== RUN   TestAccReadReplicaInstance_withEpsId
=== PAUSE TestAccReadReplicaInstance_withEpsId
=== CONT  TestAccReadReplicaInstance_basic
=== CONT  TestAccReadReplicaInstance_withEpsId
--- PASS: TestAccReadReplicaInstance_withEpsId (1297.24s)
--- PASS: TestAccReadReplicaInstance_basic (2349.45s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rds       2528.907s
```

### CODE REVIEW

File: docs/resources/rds_read_replica_instance.md [hunk 1]
```diff
@@ -70,47 +52,87 @@ The following arguments are supported:
...
-* `flavor` - (Required, String) Specifies the specification code.
-
 * `primary_instance_id` - (Required, String, ForceNew) Specifies the DB instance ID, which is used to create a read
   replica. Changing this parameter will create a new resource.
 
 * `volume` - (Required, List, ForceNew) Specifies the volume information. Structure is documented below. Changing this
   parameter will create a new resource.
 
+* `db` - (Optional, List, ForceNew) Specifies the database information. Structure is documented below. Changing this
+  parameter will create a new resource.
```
Comment (Lance52259) [inline docs/resources/rds_read_replica_instance.md]:
```suggestion
* `db` - (Optional, List, ForceNew) Specifies the database information. [Structure](xxxxxxx) is documented below. Changing this
  parameter will create a new resource.
```

File: docs/resources/rds_read_replica_instance.md [hunk 2]
```diff
@@ -70,47 +52,87 @@ The following arguments are supported:
...
+* `flavor` - (Required, String) Specifies the specification code.
+
+* `security_group_id` - (Optional, String) Specifies the security group which the read replica instance belongs to.
+
+* `fixed_ip` - (Optional, String) Specifies an intranet floating IP address of read replica instance.
+
+* `ssl_enable` - (Optional, Bool) Specifies whether to enable the SSL for read replica instance.
+
+* `parameters` - (Optional, List) Specify an array of one or more parameters to be set to the read replica instance
+  after launched. You can check on console to see which parameters supported. Structure is documented below.
```
Comment (Lance52259) [inline docs/resources/rds_read_replica_instance.md]: ditto

File: huaweicloud/services/rds/resource_huaweicloud_rds_instance.go [hunk 3]
```diff
@@ -1086,6 +949,97 @@ func buildRdsInstanceParameters(params *schema.Set) instances.ModifyConfiguratio
...
+				}
+				break
+			}
+		}
+		if restart {
+			break
+		}
+	}
+	return restart, nil
+}
```
Comment (Lance52259) [inline huaweicloud/services/rds/resource_huaweicloud_rds_instance.go]:
```suggestion
	for _, parameter := range parameters {
		name := parameter.(map[string]interface{})["name"]
		for _, v := range configs.Parameters {
			if v.Name == name && v.Restart {
				return true， nil
			}
		}
	}
	return false, nil
}
```

**Comment by Lance52259** [comment]
too many things that you done in this commit

File: huaweicloud/services/rds/resource_huaweicloud_rds_instance.go [hunk 4]
```diff
@@ -1086,6 +949,97 @@ func buildRdsInstanceParameters(params *schema.Set) instances.ModifyConfiguratio
...
+	})
+	if err != nil {
+		return fmt.Errorf("error rebooting for RDS instance (%s): %s", instanceID, err)
+	}
+
+	// wait for the instance state to be 'ACTIVE'.
+	stateConf := &resource.StateChangeConf{
+		Target:       []string{"ACTIVE"},
+		Refresh:      rdsInstanceStateRefreshFunc(client, instanceID),
+		Timeout:      d.Timeout(schema.TimeoutDefault),
```
Comment (Lance52259) [inline huaweicloud/services/rds/resource_huaweicloud_rds_instance.go]: Why is default？

File: docs/resources/rds_read_replica_instance.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/rds_read_replica_instance.md]: done

File: docs/resources/rds_read_replica_instance.md (hunk 2 above)
Comment (houpeng80) [reply docs/resources/rds_read_replica_instance.md]: done

File: huaweicloud/services/rds/resource_huaweicloud_rds_instance.go (hunk 3 above)
Comment (houpeng80) [reply huaweicloud/services/rds/resource_huaweicloud_rds_instance.go]: it has been fixed in other PR

File: huaweicloud/services/rds/resource_huaweicloud_rds_instance.go (hunk 4 above)
Comment (houpeng80) [reply huaweicloud/services/rds/resource_huaweicloud_rds_instance.go]: it has been fixed in other PR

**Pull request merged**